apache-spark-worker/build/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu
+
+MAINTAINER Frank Austin Nothaft, fnothaft@berkeley.edu
+
+RUN apt-get update && \
+  apt-get install -y \
+    python \
+    libnss3 \
+    openjdk-8-jdk \
+    curl
+
+COPY download.sh /home/
+RUN sh /home/download.sh
+
+WORKDIR /opt/apache-spark
+RUN sed -i "s/<joda.version>2.5<\/joda.version>/<joda.version>2.9.4<\/joda.version>/" pom.xml
```

Comment:
Actually, hang on. This is a POM property, right? Can't you override that with a settings.xml or on the command line? 

In lieu of that, I'm leaning towards the `git clone …; git checkout …; git apply …` proposed by @fnothaft. Even if it takes longer.